src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/MinMax.txt

```diff
@@ -0,0 +1,50 @@
+﻿>> Max(1,2)
+2
+
+>> Max(Blank(), 100, -5, "2000", Value("2001"), Text(2002))
+2002
+
+>> Max("200", "2000")
+2000
+
+>> Max(DateTimeValue("November 10, 2014 1:50 PM"), DateTimeValue("November 11, 2014 1:50 PM"))
+11/11/2014 01:50:00 PM
+
+>> Max(DateTimeValue("November 10, 2014 1:50 PM"), DateTimeValue("November 10, 2014 2:50 PM"))
+11/10/2014 02:50:00 PM
+
+>> Max(Time(1,50,0),Time(2,0,0))
+02:00:00
+
+>> Min(1,2)
+1
+
+>> Min(Date(1985,3,5),Date(1986,10,19))
+Date(1985,3,5)
+
+>> Min(-100, If(true, -1000, 200))
+-1000
+
+>> Min([1, 2, 3, 4, 5], Value)
+1
+
+>> Min([1, 2, 1/0, 4, 5], Value)
+#Error
```

Comment:
`#Error(Kind=Div0)` - we can test for the error kind now

Reply:
Added